drill/drill.sh

```diff
@@ -228,7 +228,7 @@ function main() {
   mkdir -p ${DRILL_HOME} && chown ${DRILL_USER}:${DRILL_USER} ${DRILL_HOME}
 
   # Download and unpack Drill as the pseudo-user.
-  sudo wget http://apache.mirrors.hoobly.com/drill/drill-${DRILL_VERSION}/apache-drill-${DRILL_VERSION}.tar.gz
+  sudo wget http://archive.apache.org/dist/drill/drill-${DRILL_VERSION}/apache-drill-${DRILL_VERSION}.tar.gz
```

Comment:
Can you use one of the links from HTTP section of this page instead?
http://www.apache.org/dyn/closer.cgi/drill/drill-1.13.0/apache-drill-1.13.0.tar.gz